### PR TITLE
Update Vanguard

### PR DESCRIPTION
**Pull request by kmpoppe**
Changed exception wording according to #2962.

### CODE REVIEW

**Comment by Carlgo11** [comment]
Do you mean that a phone number is required for 2FA?

**Comment by Carlgo11** [comment]
@kmpoppe ^

**Comment by kmpoppe** [comment]
@Carlgo11 I just took what was in #2962 and put it in a new PR because the other one was so garbled. As vanguard.com always seems to be in maintenance mode when I want to check back on the documentation I can't get any further :-/

**Comment by kmpoppe** [comment]
@Carlgo11 After the site maintenance (looks like saturday nigth in the US) is over, I've got further information:
- From https://investor.vanguard.com/security/security-keys
	- Do I need to register for **security codes** if I have a security key?
		- **Yes**, because if you don't have your key or if you're logging on from a device that isn't compatible, we'll need to send you a security code instead.

- From https://investor.vanguard.com/security/security-codes
	- What are **security codes** and how do they work?
		- A **security code** is a unique, single-use, 6-digit number you receive by **text** or **automated phone** call that you'll enter when you log on to your Vanguard account from a device we don't recognize as one you regularly use.

File: _data/investing.yml [hunk 1]
```diff
@@ -185,7 +185,7 @@ websites:
       phone: Yes
       hardware: Yes
       exceptions:
-        text: "Hardware 2FA requires using a Chrome browser and only supports the following hardware keys: Yubikey 4 Series, YubiKey 5 Series, Yubikey Security Key Series."
+        text: "SMS/Phone require for 2FA. Hardware 2FA requires using a Chrome browser and only supports the following hardware keys: Yubikey 4 Series, YubiKey 5 Series, Yubikey Security Key Series."
```
Comment (kmpoppe) [inline _data/investing.yml]:
```suggestion
        text: "SMS/Phone 2FA activation required before hardware 2FA can be used. Hardware 2FA requires using a Chrome browser and only supports the following hardware keys: Yubikey 4 Series, YubiKey 5 Series, Yubikey Security Key Series."
```

File: _data/investing.yml [hunk 2]
```diff
@@ -185,7 +185,7 @@ websites:
       phone: Yes
       hardware: Yes
       exceptions:
-        text: "Hardware 2FA requires using a Chrome browser and only supports the following hardware keys: Yubikey 4 Series, YubiKey 5 Series, Yubikey Security Key Series."
+        text: "SMS/Phone 2FA activation required before hardware 2FA can be used. Hardware 2FA requires using a Chrome browser and only supports the following hardware keys: Yubikey 4 Series, YubiKey 5 Series, Yubikey Security Key Series."
```
Comment (Carlgo11) [inline _data/investing.yml]:
```suggestion
        text: "Hardware based 2FA requires SMS / Phone call 2FA as a backup. Hardware 2FA is only supported on Chrome browsers and works with the following hardware keys: Yubikey 4 Series, YubiKey 5 Series, Yubikey Security Key Series."
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — DISMISSED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**